addons/binding/org.openhab.binding.modbus/ESH-INF/thing/bridge-poller.xml

```diff
@@ -42,6 +42,14 @@
 				<default>3</default>
 				<description>Number of tries when reading data, if some of the reading fail. For single try, enter 1.</description>
 			</parameter>
+			<parameter name="cacheMillis" type="integer" min="0">
+				<label>Cache duration</label>
```

Comment:
Please add a unit, i.e. ms here.

Reply:
Fixed. Found some other labels missing the unit as well.